chapters/VK_EXT_debug_utils.txt

```diff
@@ -42,7 +42,14 @@ fname:vkSetDebugUtilsObjectNameEXT as defined below.
 --
 include::{generated}/api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
-  * pname:device is the device that created the object.
+  * pname:device is the name of a valid slink:VkDevice which is related to the object in
+    pname:pNameInfo.
```

Comment:
Should say what the parameter is directly, not what its relation to other things is. I would keep the cookie-cutter original, because what it is is a dispatch parameter as usual for all the commands...

Reply:
First, these are very good points. 

Background: (which you probably already know but I'll put here for those who don't).  The original design simply copied the VK_EXT_debug_marker functionality but into a common extension with the old VK_EXT_debug_report to ease sharing of types and functionality which was proving difficult in the original two extensions.  In addition to expanding additional output info (like the object types).  Originally, it was believe the developers wouldn't need this functionality for instance and physical devices. 

The only other thing we could do here if we don't want to relax this as the modifications are allowing is to create a brand new extension that would allow debug labels for Instance and physical device. 

Personally, I'm ok with allowing the tracking of instance and physical device data within this entrypoint, but I think you're point of clearly definnig the lifecycle is important and it should be noted somewhere any instance/physical device tracking will go away with the device.  Or that it goes away with the destruction of the messenger.  My concern is that if we allow it for the messenger it could open more cans of worms (like if different devices labelled the instance differently).